app/screens/channel/channel.ios.js

```diff
@@ -70,6 +70,7 @@ export default class ChannelIOS extends ChannelBase {
                     <ChannelLoader
                         height={height}
                         style={channelLoaderStyle}
+                        spacing={true}
```

Comment:
Instead of adding an extra prop for this, we already pass in the style prop which can be used to add the margin to the bottom. That also saves us from needing an extra `View` in the `ChannelLoader` as well.

This component already looks for iPhone X as well, so we can probably just add a `marginBottom` to `style.iOSHomeIndicator` without making any changes to `ChannelLoader` or any code on Android

Reply:
@hmhealey Margin bottom does not work on the wrapper view, the other view seems to be needed to give the margin bottom.

We also do not want to give the margin to all loading pages, only the one in the post list, as we do not want that extra space at the bottom when the post textarea is not there.

Reply:
As mentioned over DM, padding works on the wrapper, and that's probably something to do with the wrapper being part of a flexbox component.

> We also do not want to give the margin to all loading pages, only the one in the post list, as we do not want that extra space at the bottom when the post textarea is not there

I meant that we can pass it in from the `ChannelIOS` screen here. It passes a screen-specific style object on the line above this, so we can add the additional padding to `channelLoaderStyle` to have it only affect this screen

Reply:
Done.